### PR TITLE
Testing:  getting repos - mocking GitHub with nock

### PR DESCRIPTION
**Pull request by saraf**
Simulate a case where a user Strider Tester is registered
with github and has admin access to TWO repositories
one which belongs to him (stridertester/proj1) and one that
belongs to a team Strider Testers Union (stridertestersunion/union-proj-1)

getRepos should return an array containing the two repositories.
we are using actual responses received from github.com - as recorded
and mocked by nock to simulate.

### CODE REVIEW

File: test/mocks/setup_nock_repos.js [hunk 1]
```diff
@@ -0,0 +1,325 @@
...
+//Set up the mocks for getting a list of repositories for the stridertest user
+//The stridertest user is an admin of two organizations 
+//    stridertestersunion and
+//    stridertesters1
+//and has two repositories to which he has admin access
+//    stridertestersunion/union-proj-1
+//    stridertester/proj1
+
+//This file sets up mock responses as received from github for each request fired
+//during the get repositories operation
```
Comment (knownasilya) [inline test/mocks/setup_nock_repos.js]: Tip: For comment blocks this big (> 2 lines) `/**/` should be used.

Comment (saraf) [inline test/mocks/setup_nock_repos.js]: Ok! Will fix.

Comment (saraf) [inline test/mocks/setup_nock_repos.js]: done.

**Comment by knownasilya** [comment]
Anyone know how to access this github user? Need to login to update the access token.

**Comment by saraf** [comment]
Hi Ilya!

Found it. Will mail it to you.

Thanks!
warm regards,
Aalhad

**Pull request merged**